src/traces/sankey/plot.js

```diff
@@ -125,13 +125,14 @@ module.exports = function plot(gd, calcData) {
 
     var linkHoverFollow = function(element, d) {
 
+        var rootBBox = gd.getBoundingClientRect();
         var boundingBox = element.getBoundingClientRect();
         var hoverCenterX = boundingBox.left + boundingBox.width / 2;
         var hoverCenterY = boundingBox.top + boundingBox.height / 2;
 
         var tooltip = Fx.loneHover({
-            x: hoverCenterX + window.scrollX,
-            y: hoverCenterY + window.scrollY,
+            x: hoverCenterX - rootBBox.left + window.scrollX,
+            y: hoverCenterY - rootBBox.top + window.scrollY,
```

Comment:
are the `window.scrollX/Y` shifts still necessary now that you're using a `getBoundingClientRect` differential?